### PR TITLE
fixed current refinements

### PR DESCRIPTION
**Pull request by HugoWith**
## CAREFUL 🚨
Are you adding a new feature/fix/style tweak ? 
If yes push to **develop branch** not on **main branch**

## Versioning
- [x] Are you sure you are pushing on the right branch?

## Objective

What:
👉 Fixing bugs on current refinement (ie: video)

https://user-images.githubusercontent.com/47173348/229879145-f13a9bca-7f04-4bcf-bf5b-0d35cde5a9b1.mov

👉 Refactor & Simplify the code 
👉 create a refinement and a cleaner way to show custom filters (through navigation state)

## Type

- [x] Bug Fix
- [ ] New Feature
- [ ] Performance Tweaks
- [ ] Style Tweaks
- [ ] Code Refactoring
- [ ] Documentation
- [ ] Tests

## Tested

✅

## Translation

- [ ] Have you translated all your Paragraphs, ButtonsTitle,... with i18n functionality?

## Documented

- [ ] Have you documented in changed file using comments
- [ ] Have you added instructions to the README if needed?

### CODE REVIEW

File: apps/boilerplate-front-end/src/components/facets/components/CurrentRefinement.jsx [hunk 1]
```diff
@@ -122,3 +96,42 @@ function CurrentRefinementGeneral(props) {
 }
 
 export default CurrentRefinements
+
+// KEEPING THIS CODE JUST IN CASE
+//____________________________________________________________________
```
Comment (matthew1809) [inline apps/boilerplate-front-end/src/components/facets/components/CurrentRefinement.jsx:101]: Why?

File: apps/boilerplate-front-end/src/components/facets/components/CurrentRefinement.jsx [hunk 2]
```diff
@@ -109,10 +70,23 @@ function CurrentRefinementGeneral(props) {
             <a
               onClick={(event) => {
                 event.preventDefault()
+
+                if (
```
Comment (matthew1809) [inline apps/boilerplate-front-end/src/components/facets/components/CurrentRefinement.jsx:74]: What is this for?

**Review by matthew1809 — COMMENTED** [review]
GG!

File: apps/boilerplate-front-end/src/components/facets/components/CurrentRefinement.jsx (hunk 1 above)
Comment (HugoWith) [reply apps/boilerplate-front-end/src/components/facets/components/CurrentRefinement.jsx:101]: Will remove it after the reviews. I just want to make sure I didn't omit a case 😊

File: apps/boilerplate-front-end/src/components/facets/components/CurrentRefinement.jsx (hunk 2 above)
Comment (HugoWith) [reply apps/boilerplate-front-end/src/components/facets/components/CurrentRefinement.jsx:74]: Navigation state back to null when removing the custom filter (accissible through the navigation bar Woman accessories even & odd)

Comment (HugoWith) [reply apps/boilerplate-front-end/src/components/facets/components/CurrentRefinement.jsx:75]: Pushed some comment for clearer explainations

**Review by VirAlex — APPROVED** [review]
Everything is fine very good work and a great find for refacto old logical 👌
LGTM

**Pull request merged**